web/src/containers/IncentivePayments.js

```diff
@@ -30,83 +29,30 @@ class IncentivePayments extends Component {
   render() {
     const { data, totals, years } = this.props;
 
-    const yearsWithColors = years.map((year, i) => {
-      const color = `bg-${COLORS[i] || 'gray'}`;
-      const colorLight = `${color}-light`;
-      return { year, color, colorLight };
-    });
-
     return (
-      <div className="table-frozen-wrapper">
-        <div className="table-frozen-scroller">
-          <table
-            className="table-cms table-fixed table-frozen-left-pane"
-            aria-hidden="true"
-          >
-            <thead>
-              <tr>
-                <th className="table-frozen-null-cell">--</th>
-              </tr>
-              <tr>
-                <th className="table-frozen-null-cell">--</th>
-              </tr>
-            </thead>
-            <tbody>
-              {INCENTIVE_ENTRIES.map(({ id, name }) => (
-                <tr key={id}>
-                  <td>{name}</td>
-                  <td>
-                    <DollarInput
-                      hideLabel
-                      wrapperClass="m0"
-                      className="fake-spacer-input m0 input input-condensed mono right-align"
-                      label="fake-spacer-input"
-                      name="fake-spacer-input"
-                    />
-                  </td>
-                </tr>
-              ))}
-            </tbody>
-          </table>
-          <table
-            className="table-cms table-fixed table-frozen-data"
-            style={{ minWidth: 1200 }}
-          >
+      <Fragment>
+        {years.map(( year ) => (
+        <Fragment>
```

Comment:
I've got a bunch of nested `Fragment`s here; I'm not sure if that's the best practice. (I also suspect maybe it's what's throwing off the test?)